lib/web_of_science/query_orcid.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module WebOfScience
+  # Use author orcid to find WOS publications for an Author
+  # fetch the wos uids for the given author
+  # e.g. WebOfScience::QueryOrcid.new(author).uids
+  class QueryOrcid
+    def initialize(author, options = {})
+      raise(ArgumentError, 'author must be an Author') unless author.is_a? Author
+
+      @orcid = author.orcidid
```

Comment:
For consistency with other orcid code, can you call this `@orcidid`?

Reply:
yup, changed